creme/creme_core/tests/views/test_mass_export.py

```diff
@@ -529,6 +544,14 @@ def test_extra_filter(self):
         # Error
         self.assertGET(400, self._build_contact_dl_url(extra_q='[123]'))
 
+    def test_backend_validator(self):
+        self.login_as_root()
+        self.assertIsNone(export_backend_registry.get_backend_class(TestExportBackend.id))
+        export_backend_registry._backend_classes[TestExportBackend.id] = TestExportBackend
+        response = self.assertGET409(self._build_contact_dl_url(doc_type=TestExportBackend.id))
+        self.assertContains(response, 'TestExportBackend.validate fail', status_code=409)
+        export_backend_registry._backend_classes = None
```

Comment:
It would be more straightforward to copy the original  `_backend_classes` value & restore the value in a `finally` block.
A better solution would be to store the registry in an attribute of the class view, to set our own registry in `setupClass()` & restore the original one in `tearDownClass()`.